cli/args/mod.rs

```diff
@@ -717,7 +717,7 @@ impl CliOptions {
 
     if let Some(env_file_name) = &flags.env_file {
       if (from_filename(env_file_name)).is_err() {
-        bail!("Unable to load '{env_file_name}' environment variable file")
+        println!("{} Environment variable file '{}' not detected", colors::yellow("Info"), env_file_name);
       }
```

Comment:
Is "Warning" more appropriate here?

Reply:
Yes, I think we should do `Warning The `--env` flag was used, but dotenv file '{}' not found.